### ISSUE TITLE
Couldn't find an existing certificate locally - not Admin

### ISSUE THREAD

**Issue by Jaime97** [fastlane/fastlane#13340]
### Question Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

##### What tool/action do you have a question about?
Cert and Sigh
##### Is this a question about documentation?
Not about the documentation itself, but related to it.
##### Is this a question about a third party plugin? (If so, please go to the plugin repository first) 
No

#### Question Description
I'm trying to generate an ipa using gym, sigh and cert. For this I am a member of a team (but not an administrator), for which I have a distribution certificate and a provisioning profile generated by an administrator. With these tools I have generated, by hand, the ipa from XCode (with which I just want to clarify that I have checked that they work correctly).

As I'm not an administrator I can't generate my own certificates and I need fastlane to find mine locally. However, it is not finding it and tries to download it, so I get an error.

I know that these issues have been addressed previously but I have not found a clear answer in them (I understand that being errors arising from signature failures are difficult to resolve). However, I have found [this issue](https://github.com/fastlane/fastlane/issues/12970). It concludes that, without being an administrator, certificates cannot be accessed even if they are installed locally. Is this true? Is there a workaround right now?

Because of the nature of my project I need to continue using cert and sigh instead of match (which I know is offered as a possible answer).

At the moment I will not provide my fastfile or my fastlane.env if I am not asked for them, so as not to lengthen the explanation too much (besides that I consider that my doubt can be resolved without having to look specifically at my particular case), but please feel free to ask if you think it is necessary.

Thanks in advance.

**Comment by Jaime97** [comment]
Yes @fastlane-bot is totally related with code signing. As I said, I didn't add the fastlane env to keep it simple and because I really think it's necessary.

**Comment by zackzachariah** [comment]
Hey @Jaime97 - I was having a similar problem on my machine. For me the issue wound up being using `cert` and `sigh` at all. Digging into the docs, I realized they are intended to *download* the certs and provisioning profiles from the apple dev portal, not *validate* and download iff missing. Given that, I wound up just deleting those calls from my `Fastfile` and everything worked perfectly!

**TLDR;** If you can build on xcode and your `Appfile` has the same `apple_id` and `team_id` defined, you should be able to build *without* `cert` and `sigh`.

**Comment by pedro380085** [comment]
Thank you @zackzachariah, that info was relevant to me.
